src/DynamoCoreWpf/Views/Core/NoteView.xaml.cs

```diff
@@ -15,6 +15,7 @@ namespace Dynamo.Nodes
 {
     public partial class NoteView : IViewModelView<NoteViewModel>
     {
+        private const double MINIMUM_ZOOM_DIRECT_NODE_EDIT = 0.5;
```

Comment:
I think this threshold is defined somewhere else, i know groups and nodeviews use it, so would be best to use the same reference as they are using if possible

Reply:
in the annotation view model the way you edit the groups tittle appears to be linked to the `GroupTextBlock.IsVisible` property.

The comment seems to point that its tied to the font size and a factor.

![image](https://user-images.githubusercontent.com/25138291/122197697-5e70df00-ce90-11eb-98e9-f1c3b4bee4fe.png)

But I am trying to find where `GroupTextBlock.IsVisible` is set to `Collapsed` or `Hidden` but dont find it

![image](https://user-images.githubusercontent.com/25138291/122198980-8ca2ee80-ce91-11eb-86d3-7205876d6a5b.png)

Reply:
yea okay, heres the converter i was thinking about https://github.com/DynamoDS/Dynamo/blob/master/src/DynamoCoreWpf/UI/Converters.cs#L1448

maybe it dosent make sense to use it...